Description: Fix man page warnings
 This is a commit pulled from CVC4's GitHub repo that fixes
 some man page warnings, combined with the corresponding change
 to the distributed non-template files that aren't in the
 repository.
.
 This can be removed once there's a new upstream release.
Author: Morgan Deters <mdeters@morgandeters.com>
Last-Update: 2014-08-05

commit c0693d3718cd40fc40ae109a7a46e6a5a6b1987e
Author: Morgan Deters <mdeters@cs.nyu.edu>
Date:   Tue Aug 5 15:59:51 2014 -0400

    Fix for manpages.

diff --git a/doc/SmtEngine.3cvc_template.in b/doc/SmtEngine.3cvc_template.in
index 3a876fe..d741fd3 100644
--- a/doc/SmtEngine.3cvc_template.in
+++ b/doc/SmtEngine.3cvc_template.in
@@ -19,6 +19,8 @@ and
 .I SmtEngine::getOption()
 use the following option keys.
 
+.ad l
+
 .RS
 .TP 10
 .I "COMMON OPTIONS"
@@ -28,6 +30,8 @@ ${remaining_manpage_smt_documentation}
 .PD
 .RE
 
+.ad b
+
 .SH VERSION
 This manual page refers to
 .B CVC4
diff --git a/doc/options.3cvc_template.in b/doc/options.3cvc_template.in
index a0d6c16..f92faf4 100644
--- a/doc/options.3cvc_template.in
+++ b/doc/options.3cvc_template.in
@@ -7,6 +7,8 @@ options \- the options infrastructure
 
 .SH AVAILABLE INTERNAL OPTIONS
 
+.ad l
+
 .RS
 .TP 10
 .I "COMMON OPTIONS"
@@ -16,6 +18,8 @@ ${remaining_manpage_internals_documentation}
 .PD
 .RE
 
+.ad b
+
 .SH VERSION
 This manual page refers to
 .B CVC4
--- cvc4-1.4.orig/doc/options.3cvc
+++ cvc4-1.4/doc/options.3cvc
@@ -7,6 +7,8 @@ options \- the options infrastructure
 
 .SH AVAILABLE INTERNAL OPTIONS
 
+.ad l
+
 .RS
 .TP 10
 .I "COMMON OPTIONS"
@@ -1264,6 +1266,8 @@ add lemmas even at regular effort
 .PD
 .RE
 
+.ad b
+
 .SH VERSION
 This manual page refers to
 .B CVC4
--- cvc4-1.4.orig/doc/SmtEngine.3cvc
+++ cvc4-1.4/doc/SmtEngine.3cvc
@@ -19,6 +19,8 @@ and
 .I SmtEngine::getOption()
 use the following option keys.
 
+.ad l
+
 .RS
 .TP 10
 .I "COMMON OPTIONS"
@@ -712,6 +714,8 @@ use the following option keys.
 .PD
 .RE
 
+.ad b
+
 .SH VERSION
 This manual page refers to
 .B CVC4
